OneSignalSDK/onesignal/src/main/java/com/onesignal/GenerateNotification.java

```diff
@@ -122,6 +130,16 @@ private static CharSequence getTitle(JSONObject fcmJson) {
    }
 
    private static PendingIntent getNewActionPendingIntent(int requestCode, Intent intent) {
+     return  getNewActionPendingIntent(false, requestCode, intent);
+   }
+
+   /**
+    * Notification delete is processed by Broadcast Receiver to avoid creation of activities that can end
+    * on weird UI interaction
+    */
+   private static PendingIntent getNewActionPendingIntent(boolean openerIsBroadcast, int requestCode, Intent intent) {
+      if (openerIsBroadcast)
```

Comment:
We can safely remove the `openerIsBroadcast` check as the open action will always be an `Activity`